astropy/stats/tests/test_circstats.py

```diff
@@ -146,3 +146,62 @@ def test_vonmisesmle():
     data = np.rad2deg(data) * u.deg
     answer = np.rad2deg(3.006514) * u.deg
     assert_equal(np.around(answer, 3), np.around(vonmisesmle(data)[0], 3))
+
+    # testing for weighted vonmisesmle
+    data = np.array(
+        [
+            np.pi / 2,
+            np.pi,
+            np.pi / 2,
+        ]
+    )  # this data has twice more np.pi/2 than np.pi
+    # get answer using astropy vonmisesmle to test
+    answer = vonmisesmle(data)
+    data_to_weigh = np.array([np.pi / 2, np.pi])
+    weights = [2, 1]
+    assert_allclose(
+        answer[0], vonmisesmle(data_to_weigh, weights=weights)[0], atol=1e-5
+    )
+    assert_allclose(
+        answer[1], vonmisesmle(data_to_weigh, weights=weights)[1], atol=1e-5
+    )
+
+    # testing for axis argument (stacking the data from the first test)
+    data = np.array(
+        [
+            [
+                3.3699057,
```

Comment:
Seems like `black` overdid it a little but this is what would make the precommit bot happy. 🤷

Reply:
I really don't like this formatting. We could manually reformat and turn off black (`fmt: off`), but let's just get this in before the freeze.